docs/websites/cms/install-and-configure-wordpress-using-wp-cli.md

```diff
@@ -0,0 +1,272 @@
+
+        source ~/.bashrc
+
+That's it. Bash Completion is enabled. To test it type `wp theme` and press **Tab**. You will see the list of available commands with `wp theme` again on the prompt.
+
+## Basics of WP-CLI
+Before getting to the main part, let us learn some essential basics of how exactly WP-CLI works. This would help you feel comfortable with what is going to follow next.
+
+So far we have seen WP-CLI is accessed through the keyword `wp`. What follows that is a set of commands and their own sub commands. For example we have a command to download WordPress which goes like `wp core download`. Here wp is the main command while core and download are its sub-commands. There can be several sub-commands for each sub-command as well.
+
+WP-CLI also comes with a pretty detailed help section which will allow you to keep tab on all the commands you would need. You can access it by typing
+
+    wp help
+
+Do that and you will get a screen similar to
+
+    wp
+
+    DESCRIPTION
+
+    Manage WordPress through the command-line.
+
+    SYNOPSIS
+
+    wp <command>
+
+    SUBCOMMANDS
+
+    cache               Manage the object cache.
+    cap                 Manage user capabilities.
+    cli                 Get information about WP-CLI itself.
+    comment             Manage comments.
+    core                Download, install, update and otherwise manage WordPress proper.
+    cron                Manage WP-Cron events and schedules.
+    db                  Perform basic database operations.
+    eval                Execute arbitrary PHP code after loading WordPress.
+    eval-file           Load and execute a PHP file after loading WordPress.
+    :
+
+
+`:` is a prompt where you need to enter few commands to navigate through this help menu. Up and down arrow keys will take you through the complete help. And typing `q` will exit the help menu. That's all you need to know for now. For complete detail on how to navigate through the complete help section, you can always type `h` at the above prompt.
+
+In our last step, we enabled Bash Completion feature for WP-CLI. To use that type `wp` and press tab twice. You would see the list of available commands. You can repeat the same by typing `wp core` and then pressing tab twice. Now you will see a list of commands that can be used with `core`.
+
+## Installing WordPress
+
+### Setting up Database
+
+1.  Before you proceed, you need to setup a database first. For that, login to the MySQL server first. Replace `user` with your MySQL username.
+
+        mysql -u user -p
+
+2.  Next step is to create a database.
+
+        create database wordpress;
+
+3.  Now we need to grant all the required privileges for the database to the mysql user.
+
+        grant all on wordpress.* to 'user' identified by 'password';
+
+4.  Type `quit` to exit the MySQL command line.
+
+### Main Install
+
+1.  Move to the Apache document root directory and give appropriate ownership to the public_html directory.
```

Comment:
You need an empty line before the code excerpt. Why would you be changing the ownership of the html document root to the user?